canonical-web-and-design/ubuntu.com
# Grid updates for /internet-of-things, /kubernetes, /kubeflow, /legal, /livepatch, /openstack and /pricing pages

## Done

Updated grid/row/col markup patterns on /internet-of-things, /kubernetes, /kubeflow, /legal, /livepatch, /openstack and /pricing pages according to point 2 of the [Vanilla 2 upgrade guide](https://ubuntu.com/blog/vanilla-framework-2-0-upgrade-guide)
- this PR doesn't address other issues caused by the upgrade, such as broken footer and stepped lists, navigation layout issues and card heading icons being misaligned.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit the following pages and ensure the grid/row/col layout does not look broken (ignoring other upgrade issues): 
- [x] /internet-of-things
- [x] /internet-of-things/appstore
- [x] /internet-of-things/contact-us
- [x] /internet-of-things/digital-signage
- [x] /internet-of-things/gateways
- [x] /internet-of-things/robotics
- [x] /internet-of-things/thank-you
- [x] /kubeflow
- [x] /kubeflow/consulting
- [x] /kubeflow/contact-us
- [x] /kubeflow/features
- [x] /kubeflow/install
- [x] /kubeflow/thank-you
- [x] /kubernetes
- [x] /kubernetes/features
- [x] /kubernetes/install
- [x] /kubernetes/managed
- [x] /kubernetes/partners
- [x] /kubernetes/thank-you
- [x] /legal
- [x] /legal/contributors
- [x] /legal/data-privacy
- [x] /legal/data-privacy/2013-03-25
- [x] /legal/terms-and-policies
- [x] /legal/ubuntu-advantage
- [x] /legal/ubuntu-advantage-service-terms
- [x] /legal/ubuntu-advantage-service-terms/ja
- [x] /legal/ubuntu-advantage-service-terms/2016-06-24
- [x] /legal/ubuntu-advantage-service-terms/2016-05-20
- [x] /legal/ubuntu-advantage-service-terms/2015-09-09
- [x] /legal/ubuntu-advantage-service-terms/2015-06-24
- [x] /legal/terms
- [x] /livepatch
- [x] /openstack
- [x] /openstack/consulting
- [x] /openstack/contact-us
- [x] /openstack/features
- [x] /openstack/install
- [x] /openstack/storage
- [x] /openstack/what-is-openstack
- [x] /pricing/consulting
- [x] /pricing/infra
- [x] /pricing/thank-you

## Issue / Card

Fixes #5676
